### PR TITLE
[REF] Ported document_page_work_instructions

### PR DESCRIPTION
**Pull request by llacroix**
Depends on `document_page_approval`. Didn't require any other changes to work. 

Depends on 
- [x] OCA/knowledge#30
- [x] #20 

Edit add depenancies (yvaucher)

### CODE REVIEW

**Comment by dreispt** [comment]
We need `document_page_approval`ported ahead of this...

**Comment by llacroix** [comment]
I'm aware of that. I used the branch that already ported `document_page_approval` to test the module. The PR is there when `document_page_approval` will be merged, it should work.

**Comment by bwrsandman** [comment]
Depends on https://github.com/OCA/knowledge/pull/25

File: document_page_work_instructions/__openerp__.py [hunk 1]
```diff
@@ -27,10 +27,15 @@
...
     """,
-    "depends": ['mgmtsystem', 'document_page_approval'],
-    "data": ['document_page_work_instructions.xml'],
+    "depends": [
+        'mgmtsystem',
+        'document_page_approval',
+    ],
+    "data": [
+        'data/document_page.xml',
+
```
Comment (dreispt) [inline document_page_work_instructions/__openerp__.py:36]: remove extra line

File: document_page_work_instructions/data/document_page.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+  <data>
```
Comment (dreispt) [inline document_page_work_instructions/data/document_page.xml]: noupdate="1"?

File: document_page_work_instructions/views/document_page.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<openerp>
+  <data>
+
+    <record model="ir.actions.act_window" id="work_instructions_action">
+      <field name="name">Work Instructions</field>
+      <field name="res_model">document.page</field>
+      <field name="view_type">form</field>
+      <field name="view_mode">tree,form</field>
+      <field name="view_id" ref="document_page.view_wiki_tree"/>
+      <field name="search_view_id" ref="document_page.view_wiki_filter"/>
```
Comment (dreispt) [inline document_page_work_instructions/views/document_page.xml]: I suggest removing these two *view_id attributes

File: document_page_work_instructions/data/document_page.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      <field name="type">content</field>
+      <field name="content">
+        <![CDATA[
+<h1>Step 1</h1>
+
+<h1>Step 2</h1>
+
+<h1>Step 3</h1>
+        ]]>
+      </field>
```
Comment (max3903) [inline document_page_work_instructions/data/document_page.xml:16]: This content can be moved in the category record above to serve as a template.

**Comment by dreispt** [comment]
Minor notes, but globally :+1:

File: document_page_work_instructions/__openerp__.py (hunk 1 above)
Comment (llacroix) [inline document_page_work_instructions/__openerp__.py:36]: Is it necessary? I wanted to separate data data from view data.

Comment (dreispt) [inline document_page_work_instructions/__openerp__.py:36]: I see. I would not add it, but I won't pick with that.

**Comment by dreispt** [comment]
Needs one more review.

**Comment by max3903** [comment]
Waiting for the content to be moved in the category record and the content record to be removed.

**Comment by llacroix** [comment]
@max3903 , @dreispt  I updated the code according to your reviews. Anything else?

**Comment by max3903** [comment]
:+1:

**Pull request merged**